olp-cpp-sdk-core/include/olp/core/client/RetrySettings.h

```diff
@@ -67,9 +67,32 @@ struct CORE_API RetrySettings {
   int max_attempts = 3;
 
   /**
-   * @brief The connection timeout limit (in seconds).
+   * @brief Maximum time for request to complete (in seconds).
+   *
+   * The default value is 90.
+   *
+   * @note Connection or data transfer will be interrupted after specified
+   * period of time ignoring `connection_timeout` and `transfer_timeout` values.
+   */
+  int timeout = 90;
```

Comment:
Can we easily change the default values of settings?

Reply:
We can left is as is (60 seconds), but then we will need to change connection value to smaller, other vice default values will violate our recomendation.

Reply:
What would be the suggested values for connection and transfer timeouts ?